lib/manager/argocd/extract.ts

```diff
@@ -0,0 +1,61 @@
+import { safeLoadAll } from 'js-yaml';
+import * as gitTags from '../../datasource/git-tags';
+import * as helm from '../../datasource/helm';
+import type { ExtractConfig, PackageDependency, PackageFile } from '../types';
+import type { ApplicationDefinition } from './types';
+import { fileTestRegex } from './util';
+
+function loadYaml(content: string): ApplicationDefinition[] {
+  return safeLoadAll(content);
+}
```

Comment:
```suggestion
function loadYaml(content: string): ApplicationDefinition[] {
  return safeLoadAll(content);
}

```

Add newline for readabillity. You can also do an inline cast so you don't need this dummy function.